AdaptiveRoads/UI/QuayRoads/QuayRoadsPanel.cs

```diff
@@ -0,0 +1,231 @@
+using ColossalFramework;
+using ColossalFramework.UI;
+using KianCommons.UI.Table;
+using KianCommons.UI;
+using static KianCommons.ReflectionHelpers;
+using static KianCommons.EnumBitMaskExtensions;
+using static KianCommons.EnumerationExtensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AdaptiveRoads.Data.QuayRoads;
+using System.Reflection;
+using KianCommons;
+using AdaptiveRoads.Manager;
+using AdaptiveRoads.Util;
+using KianCommons.UI.Helpers;
+
+namespace AdaptiveRoads.UI.QuayRoads {
+    class QuayRoadsPanel : UIDraggableWindow<QuayRoadsPanel> {
```

Comment:
maybe add an additional abstraction layer for a window with a table of properties. That might e.g. contain most of the reflection stuff done here. Then it could also be used e.g. to replace the lanes editor.